scio-avro/src/test/scala/com/spotify/scio/avro/types/TypeProviderTest.scala

```diff
@@ -761,13 +761,16 @@ class TypeProviderTest extends FlatSpec with Matchers {
 
   @AvroType.fromSchemaFile(
     """
+      |https://raw.githubusercontent.com/spotify/scio/master/
       |scio-avro/src/test/avro/
       |scio-avro-test.avsc
     """.stripMargin)
   class FromResourceMultiLine
 
-  @AvroType.fromSchemaFile("scio-avro/src/test/avro/scio-avro-test.avsc")
+  // scalastyle:off line.size.limit
+  @AvroType.fromSchemaFile("https://raw.githubusercontent.com/spotify/scio/master/scio-avro/src/test/avro/scio-avro-test.avsc")
```

Comment:
Not a huge fan of these changes since it now requires network connection to compile but that seems the only way to avoid relative paths?